JSQMessagesViewController/Controllers/JSQMessagesViewController.h

```diff
@@ -168,6 +168,12 @@
  */
 @property (assign, nonatomic) CGFloat topContentAdditionalInset;
 
+/**
+* Sets the max height of the input toolbar.
+* Set to 0 to disable max height (default)
```

Comment:
Docs: following existing format. I suggest using [VVDocumenter](https://github.com/onevcat/VVDocumenter-Xcode).

should read:

> Specifies the maximum height of the input toolbar. 
>
>The default value is `NSNotFound`, which specifies no maximum height.